package.json

```diff
@@ -63,7 +63,7 @@
     "babel-preset-react": "^6.24.1",
     "babel-preset-stage-0": "^6.24.1",
     "babel-runtime": "^6.23.0",
-    "cross-env": "^5.0.0",
+    "cross-env": "^5.1.3",
```

Comment:
Sorry, another nit; we’d rather update dependencies in separate PRs or commits instead of mixing these changes into unrelated PRs

Sorry, but thanks for suggesting the upgrade :)